server/api/api/endpoints/alerts.py

```diff
@@ -20,6 +20,7 @@
 from sqlalchemy.orm import Session
 
 import mlrun.common.schemas
+import mlrun.common.schemas.alert
```

Comment:
This seems redundant - is this a leftover? This is the only change in this file, so not sure why it's needed.